annotation-web/src/main/java/eu/europeana/annotation/web/service/controller/jsonld/BaseJsonldRest.java

```diff
@@ -590,13 +574,13 @@ protected ResponseEntity<String> enableAnnotation(String identifier, Authenticat
 	    String eTagOrigin = generateETag(storedAnno.getGenerated(), WebFields.FORMAT_JSONLD, apiVersion);
 	    checkIfMatchHeader(eTagOrigin, request);
 
-	    getAnnotationService().enableAnnotation(storedAnno.getAnnotationId());
+	    getAnnotationService().enableAnnotation(storedAnno.getIdentifier());
 
 	    MultiValueMap<String, String> headers = new LinkedMultiValueMap<String, String>(5);
 	    headers.add(HttpHeaders.VARY, HttpHeaders.ACCEPT);
 	    
-	    JsonLd annotationLd = new AnnotationLdSerializer(storedAnno);
-	    String jsonLd = annotationLd.toString(4);
+	    annotationLdSerializer.setAnnotation(storedAnno);
```

Comment:
annotationSerilizer is statefull, therefore a new object must be created for each request